src/braket/circuits/circuit.py

```diff
@@ -613,7 +613,7 @@ def add_circuit(
         if target_mapping and target is not None:
             raise TypeError("Only one of 'target_mapping' or 'target' can be supplied.")
         elif target is not None:
-            keys = sorted(circuit.qubits)
+            keys = circuit.qubits
```

Comment:
would this be a user-facing behavior change? why is it needed?

Reply:
I don't recall the testing I had done here so for now, I have removed it until retested.

Reply:
Awesome, thank you